.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Yeah, it's time to do this I think. Will make the lives of goland developers better.